app/src/main/java/com/fastlink/zekrapp/data/data/CategoryData.kt

```diff
@@ -0,0 +1,935 @@
+package com.fastlink.zekrapp.data.data
+
+val categoryData : ArrayList<ArrayList<Any>> = arrayListOf(
```

Comment:
Instead of having array list of array list you could have a data class representing an item.